pkg/virt-launcher/notify-client/notify_test.go

```diff
@@ -104,7 +104,9 @@ var _ = Describe("Notify", func() {
 				mockDomain.EXPECT().GetXMLDesc(gomock.Eq(libvirt.DomainXMLFlags(0))).Return(string(x), nil)
 				mockDomain.EXPECT().GetMetadata(libvirt.DOMAIN_METADATA_ELEMENT, "http://kubevirt.io", libvirt.DOMAIN_AFFECT_CONFIG).Return(`<kubevirt></kubevirt>`, nil)
 
-				eventCallback(mockCon, util.NewDomainFromName("test", "1234"), libvirtEvent{Event: &libvirt.DomainEventLifecycle{Event: event}}, client, deleteNotificationSent, nil, nil)
+				domainCache := util.NewDomainFromName("test", "1234")
+				eventCallback(mockCon, domainCache, libvirtEvent{Event: &libvirt.DomainEventLifecycle{Event: event}},
```

Comment:
I do not understand how the line got split here.
I suggest you either put each argument on a separate line or all in one line.
These are the options that usually fit:
```
foo(a, b, c)
foo(
  a, b, c,
)
foo(
  a,
  b,
  c,
)
```
I am surprised that `gofmt` is not taking care of this.
```